src/model/Suite.model.js

```diff
@@ -1,17 +1,22 @@
 export default class Suite {
-  constructor(name, user, host, port) {
+  constructor(id, name, owner, host, port) {
+    this.id = id;
     this.name = name;
-    this.user = user;
+    this.owner = owner;
```

Comment:
Not sure why but I used `user` for a `Suite` (which will be renamed to Workflow in another change #73 ).